src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategy.kt

```diff
@@ -899,7 +914,7 @@ abstract class ManagedStrategy(
                         } else {
                             params
                         }
-                        beforeMethodCall(owner, currentThread, codeLocation, className, methodName, params)
+                        beforeMethodCall(owner, currentThread, codeLocation, 0, className, methodName, params)
```

Comment:
This is a "Stub" as we don't actually need methodId in case of atomic method call as there can't be any other trace points -> no possible recursive spin cycles. So it's just a workaround caused by beforeMethodCall is taking `methodId` parameter.

Reply:
Let's maybe extract it into a constant, like: 
```
const val ATOMIC_METHOD_ID = 0
```

and add a comment explaining why it is always zero